app/controllers/api/v1/followed_tags_controller.rb

```diff
@@ -3,11 +3,11 @@
 class Api::V1::FollowedTagsController < Api::BaseController
   TAGS_LIMIT = 100
 
-  before_action -> { doorkeeper_authorize! :follow, :read, :'read:follows' }, except: :show
```

Comment:
I'm not sure I understand how this change relates.

Reply:
there is no :show, there is only :index, so it's basically cleaning up an unused call